### PR TITLE
WIP! Scheduler / Simplic.Data / Simplic.Data.Sql

### PR DESCRIPTION
**Pull request by simplicbe**
I've checked the following points:

- [ ] I've formatted the code by pressing (STRG + K + D)
- [ ] I've removed unnecessary usings
- [ ] I've not used magic variables
- [x] I've used localization service instead of text
- [ ] All `public` things have comments
- [ ] This has been followed: [conventions](https://simplic.github.io/dev/csharp/getting_started/conventions.html)

### CODE REVIEW

File: simplic-data/Simplic.Data.Sql/SqlRepositoryBase.cs [hunk 1]
```diff
@@ -40,21 +40,35 @@ public SqlRepositoryBase(ISqlService sqlService, ISqlColumnService sqlColumnServ
...
+        }
+
+        /// <summary>
+        /// Get a model instance from the database, using a custom column name
+        /// </summary>
+        /// <typeparam name="T">Id type</typeparam>
+        /// <param name="columnName">Column name</param>
+        /// <param name="id">Id value</param>
+        /// <returns>Model if exists</returns>
+        protected TModel GetByColumn<T>(string columnName, T id)
```
Comment (yvz5) [inline simplic-data/Simplic.Data.Sql/SqlRepositoryBase.cs]: this would not work. this custom column could have a type other than T

Comment (simplicbe) [reply simplic-data/Simplic.Data.Sql/SqlRepositoryBase.cs]: That is way the method is protected. E.g. if you want to implement something like `GetByname()`

**Comment by simplicbe** [comment]
@yvz5 but it's not really a repository.

**Comment by yvz5** [comment]
> @yvz5 but it's not really a repository.

it is a base repository, does all the things a repository does. but I would like to have it under `Simplic.Sql` so I can inherit from it without having to reference another package

**Comment by simplicbe** [comment]
@yvz5 we need to do something with this PR. Renaming may cause some problems...

**Pull request merged**